sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Common/RuleTest.cs

```diff
@@ -125,17 +125,6 @@ public void SonarDiagnosticAnalyzer_IsUsedInAllRules()
             }
         }
 
-        [TestMethod]
```

Comment:
Could we have a _OnlyRulesInSonarWarAreActivatedByDefault_ test? (I guess the utility rules might be a special case)